bindings/wasm/examples/src/explorer-did.js

```diff
@@ -0,0 +1,243 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    KeyPair,
+    KeyType,
+    Client,
+    Config,
+    Document,
+    MethodScope,
+    Service,
+    Timestamp,
+    VerificationMethod,
+    Network, ExplorerUrl
+} from '@iota/identity-wasm';
+
+/**
+ * This example shows a basic introduction on how to create a basic DID Document and upload it to the Tangle
+ * using the Account.
+ */
+(async function () {
+    const MAINNET = Network.mainnet();
+    const EXPLORER_MAINNET = ExplorerUrl.mainnet();
+
+    /* @type {{network: Network, explorer: ExplorerUrl}} */
+    const CLIENT_CONFIG = {
+        network: MAINNET,
+        explorer: EXPLORER_MAINNET,
+    }
+
+
+    // Create a default client configuration from the parent config network.
+    const config = Config.fromNetwork(CLIENT_CONFIG.network);
+
+    // Create a client instance to publish messages to the Tangle.
+    const client = Client.fromConfig(config);
+
+    // Generate a new ed25519 public/private key pair.
+    const key = new KeyPair(KeyType.Ed25519);
+
+    // Create a DID Document (an identity) from the generated key pair.
+    const doc = new Document(key, CLIENT_CONFIG.network.toString());
```

Comment:
The `CLIENT_CONFIG` is unnecessary, as is `EXPLORER_MAINNET`.

Not sure if we need to describe each line? This is not an introductory example linked in the README.md.

```suggestion
    // Use the Mainnet Tangle network.
    const network = Network.mainnet();

    // Create a default client configuration for the network.
    const config = Config.fromNetwork(network );

    // Create a client instance to publish messages to the Tangle.
    const client = Client.fromConfig(config);

    // Generate a new ed25519 public/private key pair.
    const key = new KeyPair(KeyType.Ed25519);

    // Create a DID Document (an identity) from the generated key pair.
    const doc = new Document(key, network.toString());
```